src/cf.data.pre

```diff
@@ -7112,11 +7112,22 @@ LOC: Config.visibleHostname
 DEFAULT: none
 DEFAULT_DOC: Automatically detect the system host name
 DOC_START
-	If you want to present a special hostname in error messages, etc,
-	define this.  Otherwise, the return value of gethostname()
-	will be used. If you have multiple caches in a cluster and
-	get errors about IP-forwarding you must set them to have individual
-	names with this setting.
+	The public domain name (FQDN) of this Proxy.
+
+	This name will be used in generated URLs sent to clients for
+	objects they need to download directly from Squid, and for other
+	generated responses (eg error pages) needing to display the proxy
+	name.
+
+	If not set, auto-detection of the system host name will be performed
+	based on the following details (in this order):
+		- rDNS name of first listening port address
+		- the return value of gethostname()
+		- the fallback name of "localhost"
+
+	If you have multiple caches in a cluster and get errors about
+	IP-forwarding you must set them to have individual names with
```

Comment:
It is a bit difficult to correctly attribute/dereference `them` in this sentence. It is not quite `multiple caches` nor it is `errors` mentioned earlier. I suggest `configure each cache with a unique visible_hostname (or unique_hostname) value`. I do not insist on this change.

Reply:
re-phrased the sentence. Is the new version agreeable for you?

Reply:
The `set each proxy to have individual names with this directive` phrase is still hard to comprehend. The expressions "to set a proxy to have something" and "to set proxy with this directive" are awkward/unusual; "individual" does not mean unique. To me, the phrase reads like poor automated translation from another language.

I do not insist on polishing this sentence further, but I still suggest using "unique" instead of "individual" and avoiding "to set a proxy" expression. FWIW, I can give you two more sketches to work with:

* give each proxy a unique name using a visible_hostname or unique_hostname directive
* uniquely name each proxy using a visible_hostname or unique_hostname directive

Reply:
This is getting more obtuse and unique_hostname already explains the situation. So replacing the whole paragraph with a far simpler "see also ..." reference.

Reply:
resolving as this is superceded by the latest changes.